File: pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/endpoints/message_actions/GetMessageActions.java
package com.pubnub.api.java.endpoints.message_actions;

import com.pubnub.api.java.endpoints.Endpoint;
import com.pubnub.api.models.consumer.message_actions.PNGetMessageActionsResult;

public interface GetMessageActions extends Endpoint<PNGetMessageActionsResult> {
    GetMessageActions channel(String channel);

    GetMessageActions start(Long start);

    GetMessageActions end(Long end);

    GetMessageActions limit(Integer limit);
}

File: pubnub-kotlin/pubnub-kotlin-api/src/commonMain/kotlin/com/pubnub/kmp/endpoints/message_actions/GetMessageActions.kt
package com.pubnub.kmp.endpoints.message_actions

import com.pubnub.kmp.PNFuture
import com.pubnub.api.models.consumer.message_actions.PNGetMessageActionsResult

/**
 * @see [PubNub.getMessageActions]
 */
interface GetMessageActions : PNFuture<PNGetMessageActionsResult> {
}